Resources/public/css/views/dashboard/dashboard.css

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-dashboardblockbasedview,
+.ez-view-dashboardblockbasedview * {
```

Comment:
it's not that I don't understand. I'm saying that this is not needed **right now** and it is still not (for now). Again, the point of this story/PR is to setup the extensibility point for the dashboard. Don't worry, you'll have time to add some CSS but only when there's something to style.